routes/index.js

```diff
@@ -22,9 +24,22 @@ router.get('/', (req, res) => {
   };
   client.get('search/tweets', params, function(error, tweets, response) {
     if (!error) {
+      if (Object.keys(tweet_texts).length > 200) {
+        tweet_texts = {};
+      }
+      
+      var new_tweets = [];
+      for (var i = 0; i < tweets.length; i++) {
+        if (tweets[i].text in tweet_texts) {
+          continue;
+        }
+        
+        tweet_texts[tweets[i].text] = 1;
+        new_tweets.push(tweets[i]);
```

Comment:
Looks like it doesn't work.

tweets is a json object documented like here:
https://developer.twitter.com/en/docs/tweets/search/api-reference/get-search-tweets

views/index.ejs contains a forEach call to iterate on tweets.statuses. You may structure the new_tweets object similar to that.